### ISSUE TITLE
Security Issue

### ISSUE THREAD

**Issue by Tangol** [WPChill/wp-smtp#2]
As reported here: https://wordpress.org/support/topic/password-sec-issue/ , the plugin (**v1.2.3**) doesn't encode the mail login information stored in the database, which can lead to massive data breaches.

This should be a high priority fix.

**Comment by cristianraiber** [comment]
@Tangol - agreed!

**Comment by Tangol** [comment]
Tested on https://github.com/WPChill/wp-smtp/commit/29b231db74540a4a218066db19bf2aa7068e3e64 and it seems to be working fine:
![test](https://user-images.githubusercontent.com/3336728/161551647-3dced7eb-4e9f-4859-a7c8-11dd7a4e9b5c.png)
Why not go live?

Also, I've noticed a minor wording issue:
![delete](https://user-images.githubusercontent.com/3336728/161551801-1595b6c1-0864-440b-8c33-0b15b41b6cf2.png)
The correct English here should be: `Delete options when deactivating this plugin.`

Thanks!

**Comment by cristianraiber** [comment]
@TeoAlex - please release this

**Comment by Tangol** [comment]
Yeah, it can be decoded with tools like this: https://www.base64decode.org/
But still, it's better than nothing.

**Comment by cristianraiber** [comment]
We could probably go with md5. Would that be OK? Do you have any other suggestions?

**Comment by Tangol** [comment]
@cristianraiber MD5 could work, with the condition that the user doesn't use common strings like `testing123` for the Username/Password, as these MD5 hashes can be easily looked up using tools like https://hashes.com/en/decrypt/hash

**Comment by cristianraiber** [comment]
Any ither suggestion is more than welcome. We could, in theory, use the same passowrd encryption system as WordPress. Md5 w/ a salt.

**Comment by mbabker** [comment]
A one-way hash (MD5, SHA1, etc.) isn't viable here.  You need something that is decryptable since you need to pass the plaintext values into the mail service.

**Comment by cristianraiber** [comment]
@mbabker - so it looks like we'll be sticking with base64 for the foreseeable future then ...
